src/allotropy/parsers/thermo_fisher_genesys30/constants.py

```diff
@@ -0,0 +1,13 @@
+""" Constants file for ThermoFisher Genesys 30 Adapter"""
+
+# Instrument software details
+GENESYS_SOFTWARE = "Genesys 30 instrument software"
+DISPLAY_NAME = "Thermo Fisher Genesys30"
```

Comment:
Any reason this does not have a space while the others do?

Reply:
The `DISPLAY_NAME` was autogenerated when we ran `hatch run scripts:create-parser thermo_fisher_genesys30 12/spectrophotometry`. We maintained consistency by keeping the adapter name as `thermo_fisher_genesys30` instead of `thermo_fisher_genesys_30`, resulting in the adapter creating the `DISPLAY_NAME` as "Thermo Fisher Genesys30." We also noticed that some adapter names in the `README.md` follow the convention of not having a space before numbers, treating them as part of the instrument name. To stay consistent, we followed this approach. Although the original instrument name is "Thermo Fisher Genesys 30," we changed the display name to "Thermo Fisher Genesys30" to align with the convention in the `README.md`. However, we kept the model name and software name as "Thermo Fisher Genesys 30" to accurately reflect the instrument.